### ISSUE TITLE
No Proposals with Thunderbird Unified Inbox folder

### ISSUE THREAD

**Issue by O-Herzog** [opto/nostalgy-xpi#250]
TB 115.6.0, Nostalgy++ 4.1.2
I just switched from handling two separate email accounts to TB Unified Folders where all files from the inboxes of the two email accounts are listed. But here I get only as a shift-s proposal <TB name of email account>/inbox for every email.
If I manage the emails in the separate email accounts, the shift-s proposals still work as advertised.

Thanks for taking care
O. Herzog

**Comment by opto** [comment]
can't test right now, but this is most likely resolved with v 4.1.4, not published yet

**Comment by opto** [comment]
is that solved by v 4.1.4?

**Comment by O-Herzog** [comment]
No, unfortunately: with 4.1.4, in the TB Unified Inbox, I always get offered only the proposal <mailbox name>/inbox for all mails.
O. Herzog
